rust-src/crypto_common/src/serialize.rs

```diff
@@ -540,18 +539,18 @@ use std::{fmt, io::Cursor};
 
 struct Base16Visitor<D>(std::marker::PhantomData<D>);
 
-pub fn base16_encode<S: Serializer, T: Serial>(v: &T, ser: S) -> Result<S::Ok, S::Error> {
+pub fn base16_encode<S: Serializer, T: Serial>(v: &T, ser: S) -> ParseResult<S::Ok, S::Error> {
```

Comment:
And the same here.

Basically ParseResult should have only a single argument. In cases where we used to use the std result we should keep using the std::Result, and no changes should be made.